src/main/kotlin/uk/gov/dluhc/printapi/mapper/aed/AnonymousSearchSummaryMapper.kt

```diff
@@ -10,13 +10,13 @@ import uk.gov.dluhc.printapi.database.entity.AnonymousElectorDocumentSummary as
 import uk.gov.dluhc.printapi.models.AedSearchSummary as AedSearchSummaryApi
 
 @Mapper(uses = [InstantMapper::class])
-interface AnonymousSearchSummaryMapper {
+abstract class AnonymousSearchSummaryMapper {
```

Comment:
Out of interest, what's the reason for this change (and the protected abstract method below)?

Reply:
Intention was to show that protected function is used within the mapper impl itself and not by any of kotlin classes written by us. 
The function `toAedSearchSummaryResponse` internally uses this protected function. With interface we couldn't mark these as protected.
Here is the autogenerated class for this Mapper 
<img width="854" alt="image" src="https://user-images.githubusercontent.com/73763010/236472548-75785077-38a0-4cf6-bbcc-88ef26e88afc.png">